v63003/u_inref/gtm8732nr.csh

```diff
@@ -0,0 +1,61 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# Test that a line length greater than 8192 bytes produces a LSEXPECTED warning
+#
+
+# Forcing MM, manually setting variables to avoid incompatibilities, rather than letting dbcreate
```

Comment:
Add a comment that MM is forced because we are testing MUPIP SET -DEFER_TIME which works only in MM database.